setup_env.bat

```diff
@@ -4,8 +4,19 @@ if %ERRORLEVEL% GTR 0 exit /B 1
 
 call ./scripts/win/install-protobuf.bat
 
-go get github.com/golang/protobuf/protoc-gen-go
-set GO111MODULE=off
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-set GO111MODULE=on
+SET GO111MODULE=on
```

Comment:
This is not needed, since it should be `on` to begin with. The previous code turned it `off` and back `on`.